src/app/test/test.component.ts

```diff
@@ -153,7 +154,11 @@ export class TestComponent implements OnInit {
         next: (response: TestResult): void => {
           report.reranReport = this.createReranReport(response);
         },
-        error: () => catchError(this.errorHandler.handleError()),
+        error: () =>
+          catchError((error: HttpErrorResponse): Observable<any> => {
+            report.error = error.message;
+            return of(error);
+          }),
```

Comment:
Just wondering, if the http call returns a 500 error from nginx for example, will it paste the entire 500 error html page in the table?

Reply:
In the error-handling service we are already using 'error.message', but I was not able to test it, so I am not 100% sure...

Reply:
We'll leave it like this for now, but eventually that should be refactored as well.